src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -741,6 +792,24 @@ public class foo<T>
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InvokeWithOpenAngleCommitSeeOnSpace()
+        {
+            var markupBeforeCommit = @"class c
+{
+        /// <$$
+        void foo() { }
+}";
+
+            var expectedCodeAfterCommit = @"class c
+{
+        /// <see$$/>
+        void foo() { }
+}";
+
+            await VerifyCustomCommitProviderAsync(markupBeforeCommit, "see", expectedCodeAfterCommit, commitChar: ' ');
```

Comment:
`CSharpCompletionCommandHandlerTests.cs` and `VisualBasicCompletionCommandHandlerTests.vb` are the right files to add commit tests to.